app/client/keybase/debug/keystore.go

```diff
@@ -108,48 +128,18 @@ func initializeDebugKeybase() error {
 	return nil
 }
 
-func fetchValidatorPrivateKeysFromK8S() (map[string]string, error) {
-	// Initialize Kubernetes client
-	config, err := rest.InClusterConfig()
-	if err != nil {
-		return nil, fmt.Errorf("failed to initialize Kubernetes config: %w", err)
-	}
-	clientset, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		return nil, fmt.Errorf("failed to initialize Kubernetes client: %w", err)
-	}
-
-	// Fetch validator private keys from Kubernetes
-	validatorKeysPairMap, err := pocketk8s.FetchValidatorPrivateKeys(clientset)
-	if err != nil {
-		return nil, fmt.Errorf("failed to fetch validator private keys from Kubernetes: %w", err)
-	}
-	return validatorKeysPairMap, nil
-}
-
-func fetchValidatorPrivateKeysFromFile() (map[string]string, error) {
-	// BUG: When running the CLI using the build binary (i.e. `p1`), it searched for the private-keys.yaml file in `github.com/pokt-network/pocket/build/localnet/manifests/private-keys.yaml`
-	// Get private keys from manifest file
-	_, current, _, _ := r.Caller(0)
-	//nolint:gocritic // Use path to find private-keys yaml file from being called in any location in the repo
-	yamlFile := filepath.Join(current, privateKeysYamlFile)
-	if exists, err := utils.FileExists(yamlFile); !exists || err != nil {
-		return nil, fmt.Errorf("unable to find YAML file: %s", yamlFile)
-	}
+// fetchValidatorPrivateKeysFromYaml fetches the validator private keys from the private-keys.yaml manifest file.
+func fetchValidatorPrivateKeysFromYaml() (map[string]string, error) {
```

Comment:
Not a strong opinion, curious about your thoughts about renaming this to `parseValidatorPrivateKeysFromEmbeddedYaml` or something similar that conveys the intent of this func. 

It would make crystal clear what's going on above where we call this.

If you agree I would update the GoDoc comment above accordingly as well (even tho if we do a good job with naming, the comment should almost sound redundant IMHO).

Reply:
yup agree, another good refactor.